### ISSUE TITLE
[Translation] The possibility to translate this gorgeous book.

### ISSUE THREAD

**Issue by xavieryao** [autotelicum/Smooth-CoffeeScript#31]
I'm in favor of CoffeeScript and I'd like to share with my newbie friends in China the beauty and strength of CoffeeScript. I really appreciate your work, for this is exactly the book my friends needs to get to know about programming. May I translate it into zh-cn?

**Comment by carlsmith** [comment]
Yes, you may. The work is openly licensed under [CC BY 3.0](http://creativecommons.org/licenses/by/3.0).
